indigo_app/views/works.py

```diff
@@ -594,6 +598,14 @@ def create_links(self, works_info, form):
                 # (they won't overwrite the existing works but the amendments will be linked)
                 self.link_amendment(info, form)
 
+    def add_extra_properties(self, works_info, extra_properties):
```

Comment:
This could then move into the bulk_creator, which makes more sense to me: that's the object which transforms rows into Work objects.